css2/Makefile

```diff
@@ -268,12 +264,11 @@ LN= ln
 	$(MKANCHDB) -r $*.html value-def- $(VALUEDB) $@
 	$(MKANCHDB) -r $*.html propdef- $(VALUEDB) $@
 
-%.png: %.eps; convert -density 96x96 $< $@
-%.png: %.ps; convert -density 96x96 $< $@
-%.png: %.idraw; convert -density 96x96 $< $@
-%.png: %.svg; convert -density 96x96 $< $@
-%.png: %.gif; convert -density 96x96 $< $@
-%.eps: %.fig; fig2dev -L eps -F -c $< $@
+%.big.png: %.svg; convert -density 96x96 $< $@
+%.big.png: %.gif; convert -density 96x96 $< $@
+
+%.png: %.big.png
+	zopflipng -y -m $< $@
```

Comment:
This should probably be made optional, though it's probably a good idea by default (to minimise the object size of the blobs committed to the repo, given they'll become part of the history forever).

Reply:
This isn't outdated.